k8s/upgrades/0.8.1-0.8.2/cstor_volume_upgrade.sh

```diff
@@ -5,7 +5,7 @@
 #                                                              #
```

Comment:
Remove `setDeploymentRecreateStrategy` function. since the current upgrade script is supported only when user need to upgrade(cstor volumes) from 0.8.1 it's not required to check the strategy type for deployments.

Remove the `setDeploymentRecreateStrategy` function calls also(Line 151 and 152).

Reply:
@sonasingh46 Let's update README.md from (openebs-charts PR#### as reference) to point it to https://github.com/helm/charts/pull/12944